packages/components/src/components/badge/badge.stories.ts

```diff
@@ -26,110 +37,56 @@ export default {
  */
 
 export const Default = {
+  name: 'Default',
   render: (args: any) => {
     return generateTemplate({ args });
   }
 };
 
 /**
- * The badge in all possible combinations of `variant` and `size`.
- */
-
-export const VariantAndSize = {
-  name: 'Variant × Size',
-  parameters: { controls: { exclude: ['variant', 'size'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'variant' },
-        y: { type: 'attribute', name: 'size' }
-      },
-      args
-    });
-  }
-};
-
-/**
- * The badge in all possible combinations of `variant` and `inverted`.
+ * Use the `size` attribute to set the badge’s size.
  */
 
-export const VariantAndInverted = {
-  name: 'Variant × Inverted',
-  parameters: { controls: { exclude: ['variant', 'inverted'] } },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        x: { type: 'attribute', name: 'variant' },
-        y: { type: 'attribute', name: 'inverted', values: [false, true] }
-      },
-      args,
-      options: {
-        templateBackgrounds: {
-          alternate: 'y',
-          colors: ['', 'rgb(var(--sd-color-primary, 0 53 142))']
-        }
-      }
-    });
-  }
+export const Size = {
+  name: 'Size',
+  render: () => html`
+    <div class="flex gap-12">
+      <sd-badge size="sm">8</sd-badge>
+      <sd-badge size="md">8</sd-badge>
+      <sd-badge size="lg">8</sd-badge>
+    </div>
+  `
 };
 
 /**
- * Use the `default` slot to add content to the badge.
+ * Use the `variant` attribute to set the badge’s variant.
  *
- * If you add icons to the slot, please make sure to account for accessibility by providing an alt-text.
+ * - `default`: Use the default variant for standard notifications.
```

Comment:
- `default`: Utilized for standard emphasis.

Reply:
Updated